### ISSUE TITLE
Version warning: Imagick was compiled against ImageMagick version 1808 but version 1809 is loaded.

### ISSUE THREAD

**Issue by yurii-zhuravlov** [shivammathur/homebrew-extensions#2585]
**Describe the bug**
After recent update imagemagick 7.1.0-62 -> 7.1.1-0 observe next warning each time, when php executes:
```
<br />
<b>Warning</b>:  Version warning: Imagick was compiled against ImageMagick version 1808 but version 1809 is loaded. Imagick will run but may behave surprisingly in <b>Unknown</b> on line <b>0</b><br />
```

**PHP versions**
PHP 8.1.16 (cli) (built: Feb 16 2023 07:22:21) (NTS)

**To Reproduce**
brew update && brew upgrade
...imagemagick 7.1.0-62 -> 7.1.1-0...
php -v
```
Warning: Version warning: Imagick was compiled against ImageMagick version 1808 but version 1809 is loaded. Imagick will run but may behave surprisingly in Unknown on line 0
PHP 8.1.16 (cli) (built: Feb 16 2023 07:22:21) (NTS)
```

**Comment by shivammathur** [comment]
@yurii-zhuravlov 
Thanks for reporting this. I have rebuilt all `imagick` formulae with the latest `Imagemagick`. Please reinstall the extension.

For example, for PHP 8.1:
```zsh
brew update
brew reinstall shivammathur/extensions/imagick@8.1
```

**Comment by adampatterson** [comment]
@shivammathur  I realize this is closed, I recently had the same issue with php 8.4 and this did fix it.


Are these equal? 
```shell
brew reinstall shivammathur/extensions/imagick@8.4
brew reinstall imagick@8.4
```

I installed PHP 8.4 yesterday and had previously installed it and the extensions so when I reinstalled PHP I think that led to this problem but I did run:

````shell
brew install php@8.4 xdebug@8.4 igbinary@8.4 msgpack@8.4 memcached@8.4 redis@8.4 imagick@8.4
```

`Imagick` was skipped and then I ran `brew upgrade  imagick@8.4` but continued to get the same error.
